.eslintrc

```diff
@@ -1,16 +1,15 @@
 {
-  "parser": "babel-eslint",
+  "parser": "@babel/eslint-parser",
```

Comment:
@jsnmrs I think we were seeing the errors with eslint 8, because babel-eslint is deprecated.

Reply:
ah, that makes sense!